Jenkinsfile

```diff
@@ -38,11 +42,9 @@ pipeline {
 
     stage('Build') {
       steps {
-        sh "python -m mkdocs build -f config/en/mkdocs.yml"
-        sh 'python -m mkdocs build -f config/style-guide/mkdocs.yml'
+        sh 'python3 -m mkdocs build -f config/en/mkdocs.yml'
         /* Temporary solution for lack of start page. */
         sh 'cp overrides/static/* generated/'
-        echo "${GIT_BRANCH}"
```

Comment:
Hey @jakubgs, where is now this `GIT_BRANCH` parameter?

Reply:
>where is now this GIT_BRANCH parameter?

What do you mean? That was just a useless `echo` leftover from some debugging.

Reply:
Also, check now, it was cached.